src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea extends FigureDraw {
```

Comment:
How the interface FigureArea can be logically connected with FigureDraw?
In this code, it means that FigureArea is a FigureDraw, but it is impossible. These two interfaces are responsible for different things